features/step_definitions/congress_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^a congress member exists$/) do
+  member = FactoryGirl.create(:congress_member)
+end
+
+Given(/^a stubbed phantomdc$/) do
+  Rails.application.config.congress_forms_url = "http://phantomdc.test"
```

Comment:
Awesome! Though it further confuses me about how exactly cucumber/capybara works...